### PR TITLE
Added initial files for eslint and workflow for github

### PR DESCRIPTION
**Pull request by anhtran-ccpo**
AT-6320

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: "12"
```
Comment (zachclark-ccpo) [inline .github/workflows/test.yml:25]: Any reason we went with node js version 12?

Comment (anhtran-ccpo) [reply .github/workflows/test.yml:25]: This is the version we use in our env for this code

**Comment by kylelaker-ccpo** [comment]
@anhtran-ccpo this looks good to me! I remember that you had some concerns about how the linting was working; have you had a chance to resolve those? If not, can we discuss what those particular questions are?

**Review by kylelaker-ccpo — APPROVED** [review]
Commented instead of approving. @anhtran-ccpo please see my previous comment and if everything here looks good to you, then go ahead and merge this.

**Comment by anhtran-ccpo** [comment]
Yeah I had it linting the whole code base vs just the random_quote or api package and i was wondering if that was correct

**Pull request merged**